karma.conf.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable no-var,prefer-arrow-callback,vars-on-top */
+
+require('babel-register');
+
+var IgnorePlugin = require('webpack').IgnorePlugin;
+var REACT013 = require('./src/version').REACT013;
+
+module.exports = function karma(config) {
+  config.set({
+    basePath: '.',
+
+    plugins: [
+      'karma-chrome-launcher',
+      'karma-firefox-launcher',
+      'karma-mocha',
+      'karma-webpack',
+      'karma-sourcemap-loader',
+    ],
+
+    customLaunchers: {
+      Chrome_travis: {
+        base: 'Chrome',
+        flags: ['--no-sandbox'],
+      },
+    },
+
+    frameworks: ['mocha'],
+
+    reporters: ['dots'],
+
+    files: [
+      'test/*.jsx',
+    ],
+
+    exclude: [
+      'test/_*.jsx',
+    ],
+
+    browsers: [
+      process.env.TRAVIS ? 'Chrome_travis' : 'Chrome',
+      'Firefox',
+    ],
+
+    preprocessors: {
+      'test/*.jsx': ['webpack', 'sourcemap'],
```

Comment:
`.js` might need preprocessing too - we use babel for every file, and `.jsx` is only for files that contain JSX